sdk/cosmosdb/cosmos/samples/v3/javascript/ItemManagement.js

```diff
@@ -21,7 +21,7 @@ const {
 logSampleHeader("Item Management");
 
 const itemDefs = JSON.parse(
-  readFileSync(path.resolve(__dirname, "./Shared/Data/Families.json"), "utf8")
+  readFileSync(path.resolve(__dirname, "../../src/Shared/Data/Families.json"), "utf8")
```

Comment:
This won't work. Only the contents of the `javascript` directory are published to docs.microsoft.com, so we need to refer to the files locally. If we need to copy files into the sample packages from the source, there is a package.json configuration option for that in `//sampleConfiguration` called `extraFiles` (which we have an example of in @azure/ai-form-recognizer).